packages/tools/test/docset.test.ts

```diff
@@ -3,26 +3,30 @@
  */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 
+import { TileDocument } from '@ceramicnetwork/stream-tile'
 import { Ed25519Provider } from 'key-did-provider-ed25519'
 import { fromString } from 'uint8arrays'
+import { DID } from 'dids'
+import KeyResolver from 'key-did-resolver'
 
 import { DocSet, publishIDXConfig, publishEncodedSignedDocSet } from '../src'
 
 describe('docset', () => {
+  jest.setTimeout(20000)
   beforeAll(async () => {
     const seed = fromString(
       '08b2e655d239e24e3ca9aa17bc1d05c1dee289d6ebf0b3542fd9536912d51ee9',
       'base16'
     )
-    await Promise.all([
-      ceramic.setDIDProvider(new Ed25519Provider(seed)),
-      publishIDXConfig(ceramic),
-    ])
+    const did = new DID({
+      resolver: KeyResolver.getResolver(),
+      provider: new Ed25519Provider(seed),
+    })
+    await Promise.all([ceramic.setDID(did), publishIDXConfig(ceramic)])
+    await did.authenticate()
```

Comment:
Does `did.authenticate()` need to be called after the other calls here? Couldn't it be called in parallel?

Reply:
You're right, but I don't think it matters much